ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-cluster-install-uninstall.yaml

```diff
@@ -18,7 +18,7 @@ resources:
       memory: 200Mi
 tests:
 - as: cluster-install-uninstall-hypershift-4-13-candidate
-  cron: '@yearly'
```

Comment:
This should not happen, if something is at lower frequency already we should not touch it.